packages/amazon-bedrock/src/bedrock-chat-prompt.ts

```diff
@@ -2,6 +2,7 @@ import { BedrockContentBlock } from './bedrock-api-types';
 
 export type BedrockMessagesPrompt = {
   system?: string;
+  isSystemCachePoint: boolean;
   messages: BedrockMessages;
 };
```

Comment:
Why do we have a `BedrockMessagesPrompt` that is different from the API types? Ideally we'd only have the API types and remove a level of indirection?

Reply:
Good suggestion, remnant from the rework, I restructured to use api-types.